src/dispatch/config.py

```diff
@@ -190,24 +183,35 @@ def __str__(self) -> str:
 INCIDENT_RESOURCE_NOTIFICATIONS_GROUP = config(
     "INCIDENT_RESOURCE_NOTIFICATIONS_GROUP", default="google-group-participant-notifications-group"
 )
+INCIDENT_RESOURCE_INVESTIGATION_SHEET_TEMPLATE = config(
+    "INCIDENT_RESOURCE_INVESTIGATION_SHEET_TEMPLATE", default="dispatch-incident-sheet-template"
+)
```

Comment:
Should we take the opportunity and prefix them with `DISPATCH` rather than `INCIDENT`, so we don't use `INCIDENT` twice and are consistent with the default value?
```suggestion
DISPATCH_RESOURCE_INCIDENT_SHEET_TEMPLATE = config(
    "DISPATCH_RESOURCE_INCIDENT_SHEET_TEMPLATE", default="dispatch-incident-sheet-template"
)
```

Reply:
I'm not actually sure these should even be config values? I'm not sure why we would allow users to change them.

Reply:
Yeah, now that you're mentioning it, I also don't see why they should be.